### ISSUE TITLE
1.2.0 LV2 versions of lsp-impulse-reverb do not load ui

### ISSUE THREAD

**Issue by novalix** [sadko4u/lsp-plugins#225]
It 's me again:-)

Here is the log message:
`Broken LV2 UI /usr/local/lib/lv2/lsp-plugins.lv2/lsp-plugins-lv2-1.2.0.so (no lv2ui_descriptor symbol found)`

gdb doesn 't tell me anything more exiting since there is no crashing involved. Ardour just loads its default ui, which carla does anyway.

The standalone and vst versions load the ui just fine.

**Comment by sadko4u** [comment]
Could reproduce on Ardour 6.5:
```
suil error: Broken LV2 UI /usr/local/lib/lv2/lsp-plugins.lv2/lsp-plugins-lv2-1.2.0.so (no lv2ui_descriptor symbol found)
suil error: Broken LV2 UI /usr/local/lib/lv2/lsp-plugins.lv2/lsp-plugins-lv2-1.2.0.so (no lv2ui_descriptor symbol found)
suil error: Broken LV2 UI /usr/local/lib/lv2/lsp-plugins.lv2/lsp-plugins-lv2-1.2.0.so (no lv2ui_descriptor symbol found)
```

Actually, I don't know the reason. Maybe @x42 or @drobilla can say what's happening.

**Comment by sadko4u** [comment]
Found the problem, thanks to @x42.
The case was wrong LV2 URI in the metadata descriptor of the Impulse Reverb plugin.
Please verify.

**Comment by novalix** [comment]
I can confirm that the ui now loads on both hosts.
Thanks to both of you.

**Comment by sadko4u** [comment]
Available in 1.2.0.
